runtime/calamari/src/lib.rs

```diff
@@ -921,6 +947,21 @@ impl_runtime_apis! {
         }
     }
 
+    impl nimbus_primitives::NimbusApi<Block> for Runtime {
+        fn can_author(author: NimbusId, relay_parent: u32, parent_header: &<Block as BlockT>::Header) -> bool{
+            System::initialize(&(parent_header.number + 1), &parent_header.hash(), &parent_header.digest);
+
+            // And now the actual prediction call
+            <AuthorInherent as nimbus_primitives::CanAuthor<_>>::can_author(&author, &relay_parent)		}
```

Comment:
Can you reformat this part of code? `}` looks weird.

Reply:
Right, dunno how that happened. Fixed it